pkg/rook/status.go

```diff
@@ -25,29 +25,41 @@ import (
 )
 
 var scriptPrintSpecificCRStatus = `
-kubectl -n %s get %s -ojson | jq --monochrome-output '.items[].status'
+kubectl -n %s get %s
 `
 
 var getCrdList = `
 kubectl -n  %s get crd | awk '{print $1}' | sed '1d'
 `
 
-func PrintCustomResourceStatus(clusterNamespace string, arg []string) {
+func PrintCustomResourceStatus(clusterNamespace string, arg []string, json bool) {
 	if len(arg) == 1 && arg[0] == "all" {
 		command := fmt.Sprintf(getCrdList, clusterNamespace)
 		allCRs := strings.Split(exec.ExecuteBashCommand(command), "\n")
 		allCRs = allCRs[:len(allCRs)-1] // remove last empty line which is not a CR
 		for _, cr := range allCRs {
 			logging.Info(cr)
+			if json {
+				cr = cr + " -o json"
```

Comment:
How about a helper method for the additional flag since it's used in multiple places?

Reply:
done.